src/api/fragments/location.generated.tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable import/no-duplicates, @typescript-eslint/no-empty-interface */
```

Comment:
This fragements folder may not make a lot of sense, esp now that less is in the api/ folder.
We don't need to change it now. But it would be good in the future to co-locate it with the `displayLocation` function and maybe move that somewhere else. At least a folder within api/ if not moving to components/.

Reply:
Noted, makes sense!